db/testing.py

```diff
@@ -11,7 +11,7 @@
 
 ADMIN_SQL_DIR = os.path.join(os.path.dirname(os.path.realpath(__file__)), '..', 'admin', 'sql')
 TEST_DATA_PATH = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'test_data')
-
+GID_TYPE_MBID = 'mbid'
```

Comment:
All of the possible types for `gid` column should be defined in one place and imported when needed.